libclc/generic/include/spirv/math/fmod.h

```diff
@@ -0,0 +1,12 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#define __SPIRV_FUNCTION __spirv_ocl_fmod
```

Comment:
I don't see definition of `__spirv_ocl_fmod` function. Is it a bug?

Reply:
There may not be a definition for `__spirv_ocl_fmod` yet. We haven't finished implementing all of the builtins yet, but have some internal patches with additional builtins in progress to raise later.